### ISSUE TITLE
Unit for scale prefix 1e-15

### ISSUE THREAD

**Issue by mcrot** [ContactEngineering/SurfaceTopography#164]
I've just got this error message while evaluating some existing data in Topobank
with a version 0.99:
```python
[2021-11-15 11:52:57,002: ERROR/ForkPoolWorker-7] Task topobank.taskapp.tasks.perform_analysis[366e2f1e-9088-46d7-844a-e5f704f2009c] raised unexpected: ValueError('Cannot find unit for scale prefix 1e-15.')
Traceback (most recent call last):
  File "/home/michael/work/imtek/topobank/venv/lib/python3.8/site-packages/celery/app/trace.py", line 450, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/home/michael/work/imtek/topobank/venv/lib/python3.8/site-packages/celery/app/trace.py", line 731, in __protected_call__
    return self.run(*args, **kwargs)
  File "/home/michael/work/imtek/topobank/topobank/taskapp/tasks.py", line 131, in perform_analysis
    result = analysis.function.eval(subject, **kwargs)
  File "/home/michael/work/imtek/topobank/topobank/analysis/models.py", line 288, in eval
    return self.get_implementation(subject_type).eval(subject, **kwargs)
  File "/home/michael/work/imtek/topobank/topobank/analysis/models.py", line 340, in eval
    return pyfunc(subject, **kwargs)
  File "/home/michael/work/imtek/topobank/topobank/analysis/functions.py", line 1182, in contact_mechanics
    unit = suggest_length_unit_for_data('linear', displacement_xy.data, topography.unit)
  File "/home/michael/work/imtek/topobank/venv/lib/python3.8/site-packages/SurfaceTopography/Support/UnitConversion.py", line 191, in suggest_length_unit_for_data
    return suggest_length_unit(scale, fac * mn, fac * mx)
  File "/home/michael/work/imtek/topobank/venv/lib/python3.8/site-packages/SurfaceTopography/Support/UnitConversion.py", line 167, in suggest_length_unit
    raise ValueError(f'Cannot find unit for scale prefix {fac}.')
ValueError: Cannot find unit for scale prefix 1e-15.
```
I suggest to just add all known prefixes.

**Comment by pastewka** [comment]
Wow. Which file is this?

**Comment by pastewka** [comment]
(This looks like there is something else that is wrong. I've added a permanent fix to PR #163)
